packages/mco/components/mco-dashboard/disaster-recovery/parsers/discovered-parser.tsx

```diff
@@ -121,10 +122,10 @@ export const useDiscoveredParser: UseDiscoveredParser = (
                   kubeObjSyncInterval:
                     drPlacementControl.spec?.kubeObjectProtection
                       ?.captureInterval,
-                  lastKubeObjectSyncTime:
-                    // ToDo: Update with correct status field which will report kube object last sync time
-                    // @ts-ignore
-                    drPlacementControl?.status?.lastKubeObjectSyncTime,
+                  // lastTransitionTime just to decided the status, it can't be considered as last synced on.
+                  lastKubeObjectSyncTime: getKubeObjectLastSyncTime(
```

Comment:
if that's the case we might want to rename here as well ??
```suggestion
                  kubeObjectLastTransitionTime: getKubeObjectLastTransitionTime(
```

Reply:
makesense